ui/hooks/usePetnamesEnabled.ts

```diff
@@ -1,9 +1,6 @@
-export function usePetnamesEnabled(): boolean {
-  let enabled = false;
-
-  ///: BEGIN:ONLY_INCLUDE_IF(petnames)
-  enabled = true;
-  ///: END:ONLY_INCLUDE_IF
+import { useStore } from 'react-redux';
+import { getPetnamesEnabled } from '../selectors';
 
-  return enabled;
+export function usePetnamesEnabled(): boolean {
+  return getPetnamesEnabled(useStore().getState());
```

Comment:
Minor, but since this is no longer static code, should we make a small unit test?

Reply:
Done.